modules/core/src/adapter/resources/shader.ts

```diff
@@ -102,8 +107,12 @@ export abstract class Shader extends Resource<ShaderProps> {
 
     const shaderName: string = getShaderInfo(this.source).name;
     const shaderTitle: string = `${this.stage} ${shaderName}`;
-    const htmlLog = formatCompilerLog(messages, this.source, {showSourceCode: 'all', html: true});
-
+    let htmlLog = formatCompilerLog(messages, this.source, {showSourceCode: 'all', html: true});
+    // Show translated source if available
+    const translatedSource = this.getTranslatedSource();
+    if (translatedSource) {
```

Comment:
I wonder if we should make this opt-in with a prop. `props.debugTranslateSource`?

Reply:
> I wonder if we should make this opt-in with a prop. props.debugTranslateSource?

Open to adding that if desired. 
- The translated shader source only shows up if 1) the WebGL extension is available, 2) the compilation succeeds (most of the time this window is shown when the compilation fails) 
- Also it is at the bottom of the window easily ignored / usually requires scrolling to even see it.